modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/display/context/LayoutsAdminDisplayContext.java

```diff
@@ -707,6 +707,8 @@ public JSONArray getLayoutsJSONArray(
 
 			layoutJSONObject.put(
 				"title", layout.getName(themeDisplay.getLocale())
+			).put(
+				"tooltip", isTooltip(layout.getName(themeDisplay.getLocale()))
```

Comment:
Just a naming issue, but I bet it would be more correct to call it isTooltipRequired